src/macros/compare.rs

```diff
@@ -0,0 +1,77 @@
+/// Run an experiment comparing the execution time of two blocks of code
+/// NOTE: Try to minimise variation in tag values (avoid things like timestamps or ids). See note in lib docs!
+#[macro_export]
+macro_rules! compare {
```

Comment:
there should be more docs here, describing how to use the macro, because the docs in their current form aren't much help